src/product/billOfLading/billOfLadingV1Carrier.ts

```diff
@@ -0,0 +1,72 @@
+
+import { StringDict } from "../../parsing/common";
+import { Polygon } from "../../geometry";
+
+/**
+ * The shipping company responsible for transporting the goods.
+ */
+export class BillOfLadingV1Carrier {
+  /** The name of the carrier. */
+  name: string | null;
+  /** The professional number of the carrier. */
+  professionalNumber: string | null;
+  /** The Standard Carrier Alpha Code (SCAC) of the carrier. */
+  scac: string | null;
+  /** Confidence score */
+  confidence: number = 0.0;
+  /** The document page on which the information was found. */
+  pageId: number;
+  /**
+   * Contains the relative vertices coordinates (points) of a polygon containing
+   * the field in the document.
+   */
+  polygon: Polygon = [];
```

Comment:
Are these fields hard-coded in the generator or are they coming from the OpenAPI spec file?

Because LLMs will not have a `page_id` or `polygon` field unless specified in the product's creation.

Reply:
It's hardcoded, since it used to be there for older APIs. I don't think it's hurt to leave here, as it'll be forwards-compatible if we enable polygons, and doesn't break anything as-is.